### DISCUSSION TITLE
Cannot connect to LDAP server in local network

### DISCUSSION THREAD

**Discussion by asemoq** [nextcloud/all-in-one#2055] [Questions] [answered]
Server url - ldap://192.168.47.22
Error in web interface - Lost connection to LDAP server.
I have checked ldap server from other application, it works
Could you help me please

**Comment by szaimen** [comment, marked as answer]
Hi, this should usually work out of the box. If not, I would check your network infrastrucutre. Can you ping other devices in your network from any container internally?

**Reply by asemoq** [reply]
I can't ping other devices from nextcloud container

**Comment by szaimen** [comment]
@asemoq is this a new AIO instance?

**Reply by asemoq** [reply]
No, I use version 1.7.1 with OnlyOffice

**Reply by szaimen** [reply]
Hm... Can you update to 4.4.1?

**Reply by asemoq** [reply]
I will lose onlyoffice 🥲

**Reply by szaimen** [reply]
No, you will not
